src/Database/Driver/Sqlserver.php

```diff
@@ -191,10 +191,6 @@ public function prepare(Query|string $query): StatementInterface
         $this->connect();
 
         $sql = $query;
-        $options = [
-            PDO::ATTR_CURSOR => PDO::CURSOR_SCROLL,
-            PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE => PDO::SQLSRV_CURSOR_BUFFERED,
```

Comment:
We still want this. This makes sqlsrv driver fetch the results instead of using a server-side buffer.

Reply:
Restored below, all green now.